routes/images.go

```diff
@@ -199,28 +201,25 @@ func (i Images) Destroy(w http.ResponseWriter, r *http.Request) {
 			if instance.ImageID != id {
 				continue
 			}
+			i.Logger.With("instance", instance.ID).Info("destroying instance")
 			err = i.InstanceStore.Destroy(instance)
 			if err == nil {
 				err = i.Executor.DestroyInstance(instance.ID)
 			}
 			if err != nil {
-				log.Print(err.Error())
+				i.Logger.With("error", err.Error()).Error("failed to destroy instance")
 				RenderError(w, http.StatusInternalServerError, internalServerError)
 				return
 			}
 		}
 	}
 
+	i.Logger.With("image", id).Error("destroying image")
```

Comment:
Why is this an `Error`? Surely it should be `Info`

Reply:
It is, my mistake.

Reply:
This mistake is actually not possible to make if your error function takes an actual `error` instead of a string.